tests/main/interfaces-framebuffer/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Ensure that the framebuffer interface works.
+
+details: |
+    The framebuffer interface allows to access the /dev/fb* buffer files.
+
+    The test uses the test-snapd-framebuffer snap to write/read from /dev/fb0.
+
+    The test also checks the interface connection and disconnection works properly.
+
+prepare: |
+    snap try $TESTSLIB/snaps/test-snapd-framebuffer
+
+execute: |
+    CONNECTED_PATTERN=":framebuffer +test-snapd-framebuffer"
+    DISCONNECTED_PATTERN="\- +test-snapd-framebuffer:framebuffer"
+
+    if [ ! -e /dev/fb0 ]; then
+        echo "Framebuffer not available on /dev/fb0"
+        exit 0
```

Comment:
Is this intended, shouldn't this be an error? Or is it available on some of the systems only? Should we restrict this test to be run on select systems instead?

Reply:
That could be done for sure, but as I was asked to detect it dinamically the path in the uhid test (which is similat to this one), I though that should be better to make this detection dinamic here.